src/altinn-app-frontend/src/features/form/containers/GroupContainer.test.tsx

```diff
@@ -244,7 +244,9 @@ describe('GroupContainer', () => {
     const store = render({ container: mockContainerInEditModeWithTrigger });
     const user = userEvent.setup();
 
-    const editButton = screen.getAllByText('Ferdig')[0].closest('button');
+    const editButton = screen
+      .getAllByText('Lagre og lukk')[0]
+      .closest('button');
```

Comment:
I think you can use something like this instead, which IMO looks a bit better:

```jsx
screen.getByRole('button', {
  name: /lagre og lukk/i
})
```

A good tool to help finding good selectors is https://testing-playground.com/ . You can use `screen.debug()` to print out the DOM to the console during the testrun, and paste it into testing-playground.com, and then click on items to get suggestions for selectors.